scripts/cut_release_branch.py

```diff
@@ -51,7 +52,8 @@ def new_version_type(arg, pattern=re.compile(r'\d\.\d\.\d')):
 
 # Construct the new branch name.
 NEW_BRANCH_NAME = 'release-%s' % TARGET_VERSION
-NEW_APP_YAML_VERSION = TARGET_VERSION.replace('-', '.')
+NEW_APP_YAML_VERSION = TARGET_VERSION.replace('.', '-')
+assert '.' not in NEW_APP_YAML_VERSION
```

Comment:
do you want this assert to stay or was this for debug?

Reply:
I'd like it to stay, I think. It's something worth double-checking.